ci-operator/step-registry/aws/c2s/init-token-service/aws-c2s-init-token-service-ref.yaml

```diff
@@ -0,0 +1,19 @@
+ref:
+  as: aws-c2s-init-token-service
+  from_image:
+    namespace: ocp
+    name: "4.12"
+    tag: upi-installer
```

Comment:
Same here - `centos8` should suffice

Reply:
This step need both `jq` and `oc`, I guess that is why we use upi-installer here.